.github/workflows/ci.yml

```
@@ -47,8 +47,8 @@ jobs:
         brew:
           - openjpeg
       envs: |
-        - windows: py310
-        - macos: py39
+        - windows: py311
+        - macos: py310
```

Is it worth keeping around a `py39` normal job instead of relying on oldestdeps? I think it is.
```suggestion
        - macos: py310
        - linux: py39
```